questions-answers-application/welcome.vb

```diff
@@ -14,4 +14,21 @@ Public Class welcome
         LoadQuestions()
     End Sub
 
+    Private Sub CreateQuestionToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CreateQuestionToolStripMenuItem.Click
+        CreateQuestion.ShowDialog()
+        LoadQuestions()
+    End Sub
+
+    Private Sub UpdateQuestionToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles UpdateQuestionToolStripMenuItem.Click
+        Dim updateQuestionForm As New UpdateQuestion(getQuestionId())
+        updateQuestionForm.ShowDialog()
+    End Sub
+
+    Public Function getQuestionId() As Integer
+        Return dgvQuestions.Item("id", dgvQuestions.CurrentRow.Index).Value
```

Comment:
You should [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up by referencing `Return getQuestionValue("id")`